Source/ACE.Database/WorldDatabase.cs

```diff
@@ -373,6 +373,23 @@ public PointsOfInterest GetCachedPointOfInterest(string name)
             return null;
         }
 
+        /// <summary>
+        /// Will return all Points of Interests from the DB
+        /// </summary>
+        public List<PointsOfInterest> GetAllPointsOfInterests()
```

Comment:
GetAllPointsOfInterests should probably add to the POI Cache, since it's pulling from the database anyway.

Reply:
The Plug-in does Eat the message so it will not go to the Chat Windows. Currently that line is commented out so I can see what is being sent.
POI Cache, I did have that, I Will add it back.